File: src/Exception/UndeclaredVariableException.java
package Exception;

/**
 * Created by vladc on 28.10.2016.
 */
public class UndeclaredVariableException extends ExpressionEvaluationException {
    public UndeclaredVariableException() {
        super();
    }
    public UndeclaredVariableException(String message) {
        super(message);
    }
}

File: Toy-Language-Interpreter_C#/Toy-Language-Interpreter/MyException/UndeclaredVariableException.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Toy_Language_Interpreter.MyException
{
    class UndeclaredVariableException : ExpressionEvaluationException
    {
        public UndeclaredVariableException() { }
        public UndeclaredVariableException(string str) : base(str) { }
        public UndeclaredVariableException(string str, Exception e) : base(str, e) { }
    }
}
